includes/widgets/traits/button-trait.php

```diff
@@ -503,6 +438,161 @@ protected function register_button_style_controls( $args = [] ) {
 		);
 	}
 
+	protected function register_icon_style_controls( $args = [] ) {
+		$default_args = [
+			'section_condition' => [],
+			'alignment_default' => '',
+			'alignment_control_prefix_class' => 'elementor%s-align-',
+			'content_alignment_default' => '',
+		];
+
+		$args = wp_parse_args( $args, $default_args );
+
+		$start = is_rtl() ? 'right' : 'left';
+		$end = is_rtl() ? 'left' : 'right';
+
+		$this->add_control(
+			'icon_align',
+			[
+				'label' => esc_html__( 'Position', 'elementor' ),
+				'type' => Controls_Manager::CHOOSE,
+
+				'options' => [
+					'row' => [
+						'title' => esc_html__( 'Start', 'elementor' ),
+						'icon' => "eicon-h-align-{$start}",
+					],
+					'row-reverse' => [
+						'title' => esc_html__( 'End', 'elementor' ),
+						'icon' => "eicon-h-align-{$end}",
+					],
+					'column' => [
+						'title' => esc_html__( 'Above', 'elementor' ),
+						'icon' => "eicon-v-align-top",
+					],
+					'column-reverse' => [
+						'title' => esc_html__( 'Below', 'elementor' ),
+						'icon' => "eicon-v-align-bottom",
+					],
+				],
+				'selectors_dictionary' => [
+					'left' => is_rtl() ? 'row-reverse' : 'row',
+					'right' => is_rtl() ? 'row' : 'row-reverse',
+					'above' => is_rtl() ? 'column-reverse' : 'column',
+					'below' => is_rtl() ? 'column' : 'column-reverse',
+				],
+				'selectors' => [
+					'{{WRAPPER}} .elementor-button-content-wrapper' => 'flex-direction: {{VALUE}};',
```

Comment:
This styling can be updated inside SCSS:

```suggestion
					'{{WRAPPER}}' => '{{VALUE}};',
```